com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -1725,6 +1725,9 @@ private List<TaskResult> doBuild(IProgress monitor, String... commands) throws T
         }
 
         boolean texture_compress = this.option("texture-compression", "false").equals("true");
+
+        System.out.println("Has texture compression? " + texture_compress);
```

Comment:
I don't think we should add more "printf" style logging, as this is not required very often.
I suggest looking into the logger that is available in most systems (including Project.java) (e.g. using Level.FINE)